docs/apple-privacy-manifest.md

```diff
@@ -0,0 +1,431 @@
+            </array>
+        </dict>
+    </array>
+</dict>
+</plist>
+```
+
+Now that the file has been created, it needs to be placed properly when building the app. The `PrivacyInfo.xcprivacy` is consider a resource when it is time to build the bundle. In accordance with [Placing Content ina Bundle](https://developer.apple.com/documentation/bundleresources/placing_content_in_a_bundle) the file is placed in the root of the Bundle. Use the proper set of instructions below for your ptoject type:
+
+## .NET MAUI
+1. Copy the `PrivacyInfo.xcprivacy` from your documents folder to the `Platforms/iOS` folder in your .NET MAUI project.
+1. In your favorite text editor, edit the .NET MAUI csproj project file.
+1. Add the following elements to the bottom of the root  `<Project>` element:
+    ```xml
+    <ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">
+      <BundleResource Include="Platforms\iOS\PrivacyInfo.xcprivacy" LogicalName="PrivacyInfo.xcprivacy" />
+    </ItemGroup>
+    ```
+    This will package the file into the iOS app at the root of the bundle. 
+
+## .NET for iOS (net?-ios) 
+1. Copy the `PrivacyInfo.xcprivacy` from your documents folder to the `Resources` folder in your .NET for iOS project. This is all that is needed to package the file into the iOS app at the root of the bundle.
+
+## .NET for tvOS (net?-tvos)
+1. Copy the `PrivacyInfo.xcprivacy` from your documents folder to the root folder of your .NET for tvOS project.
+1. In your favorite text editor, edit the .NET for tvOS csproj project file.
+1. Add the following elements to the bottom of the root  `<Project>` element:
+    ```xml
+    <ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'tvos'">
+      <BundleResource Include="PrivacyInfo.xcprivacy" LogicalName="PrivacyInfo.xcprivacy" />
+    </ItemGroup>
+    ```
+    This will package the file into the tvOS app at the root of the bundle. 
+
+## Xamarin iOS including Xamarin.Forms
+1. Copy the `PrivacyInfo.xcprivacy` from your documents folder to the root folder of your Xamarin.iOS project.
+1. In your favorite text editor, edit the Xamarin.iOS csproj project file.
+1. Locate the `<ItemGroup>` that contains other `<BundleResource>` elements and add the following element:
+    ```xml
+    <BundleResource Include="PrivacyInfo.xcprivacy" LogicalName="PrivacyInfo.xcprivacy" />
+    ```
+    This will package the file into the iOS app at the root of the bundle. 
+
+
+Once added to your project, the `PrivacyInfo.xcprivacy` file will need to be updated if there are any additional API usages from additional categories or additional reasons for usage. This will include adding a NuGet package or Binding project that calls into any of Apple’s [Required Reason APIs][RequiredReasonAPI]. It is ultimately yuour responsibility to provide an accurate `PrivacyInfo.xcprivacy` file, failing to do so may result in the App Store rejecting your submission.
```

Comment:
```suggestion
Once added to your project, the `PrivacyInfo.xcprivacy` file will need to be updated if there are any additional API usages from additional categories or additional reasons for usage. This will include adding a NuGet package or Binding project that calls into any of Apple’s [Required Reason APIs][RequiredReasonAPI]. It is ultimately your responsibility to provide an accurate `PrivacyInfo.xcprivacy` file, failing to do so may result in the App Store rejecting your submission.
```